decidim-core/app/cells/decidim/card_metadata_cell.rb

```diff
@@ -53,6 +53,56 @@ def space_item
       }
     end
 
+    def start_date
+      nil
+    end
+
+    def end_date
+      nil
+    end
+
+    def dates_blank?
+      [start_date, end_date].any?(&:blank?)
+    end
+
+    def current_date
+      @current_date ||= Date.current.to_time
```

Comment:
Maybe you could use DateTime  ?

Reply:
There is a [rubocop setting](https://github.com/decidim/decidim/blob/aa2968e1fc3d5514d3c6fcc49b0a099611b7ae4a/.rubocop_ruby.yml#L825-L826) that prevents the use of DateTime